### ISSUE TITLE
Update the TikTok embed handler to use `amp-tiktok` component

### ISSUE THREAD

**Issue by pierlon** [Automattic/amp-wp#6060]
## Feature description

Now that https://github.com/ampproject/amphtml/pull/32651 has been merged we can begin the process of updating [`AMP_TikTok_Embed_Handler`](https://github.com/ampproject/amp-wp/blob/b94e5e8595cc2326ee094130c04a8575f3e49918/includes/embeds/class-amp-tiktok-embed-handler.php) to use the `amp-tiktok` component instead of `amp-embedly-card`.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* Inserting a TikTok embed should result in the `amp-tiktok` component being used on the AMP page.

## Implementation brief

* Update the `AMP_TikTok_Embed_Handler` to use the `amp-tiktok` component rather than the `amp-embedly-card` one.

## QA testing instructions

* 

## Demo

* 

## Changelog entry

*

**Comment by westonruter** [comment]
Looks like we're still waiting on https://github.com/ampproject/amphtml/issues/32219 to be closed. The validator spec hasn't yet been updated.

**Comment by westonruter** [comment]
This can now be worked on as a PR off of #6436.

**Comment by westonruter** [comment]
> This can now be worked on as a PR off of #6436.

The PR has been merged into `develop`. This work can proceed.

**Comment by delawski** [comment]
**QA Passed**

I've pasted a TikTok link into a post that got automatically converted into an embed. I've then visited the AMP version of the post and confirmed that - as expected - the `amp-tiktok` element has been used to render the embed:

<img width="969" alt="Screenshot 2021-08-27 at 15 55 25" src="https://user-images.githubusercontent.com/478735/131138547-6eeae025-4e96-4cca-9b5d-6e4c58609e14.png">

*Tested against Version 2.2.0-alpha-20210825T212353Z-ab5a1e7*

**Comment by westonruter** [comment]
(To re-check on 2.1.4)

**Comment by westonruter** [comment]
I've fixed the 2.1 build errors via https://github.com/ampproject/amp-wp/pull/6574. That should allow a new 2.1 branch build to be created, although there is a JS unit test error still. See https://github.com/ampproject/amp-wp/issues/6235#issuecomment-907332957.

**Comment by delawski** [comment]
Confirmed to be working as expected on 2.1.4-alpha:

<img width="1527" alt="Screenshot 2021-08-30 at 18 03 58" src="https://user-images.githubusercontent.com/478735/131369238-d05ff419-f202-4ea7-b38e-ce7072dac4b0.png">
